rules-tests/CodingStyle/Rector/Catch_/CatchExceptionNameMatchingTypeRector/Fixture/skip_change_variable_next_jump_try.php.inc

```diff
@@ -1,20 +0,0 @@
-<?php
-
-namespace Rector\Tests\CodingStyle\Rector\Catch_\CatchExceptionNameMatchingTypeRector\Fixture;
-
-class SkipChangevariableNextJumpTry
-{
-    public function run()
-    {
-        if (rand(0, 1)) {
-            try {
-            } catch (SomeException $typoException) {
```

Comment:
I will create a PR for it :)

Reply:
Nested try/catch should be handled too. 

I think this would cause a bug even for those on top. As we do not handle rename after. 

If you find a better way without parent node, go for it :+1:  Maybe custom node traversers is needed.

Reply:
https://github.com/rectorphp/rector-src/pull/3931